### ISSUE TITLE
Create code snippets, part 7 (follow-up for PR)

### ISSUE THREAD

**Issue by josmar-crwdstffng** [angular/angular#43526]
### Description

Create or generate code-examples that align with the new step added in the #43500 PR.
The step lacks code-examples with the following details.

app.module.ts

```typescript
declarations: [
  AppComponent, TopBarComponent, ProductListComponent,
  ProductDetailsComponent
],
```
app.module.ts

```typescript
import { ProductDetailsComponent } from ‘./product-details/product-details.component’;
```

### What is the affected URL?

start/start-routing

### Please provide the steps to reproduce the issue

n/a

### Please provide the expected behavior vs the actual behavior you encountered

Please add a new code snippet to support the following step that is newly added to the [Associate a URL path with a component](https://angular.io/start/start-routing#associate-a-url-path-with-a-component) section on the [Adding navigation](https://angular.io/start/start-routing) topic.

```md
To make ProductDetailsComponent available to other components in the application, add it to AppModule's declarations in app.module.ts.
```

### Please provide a screenshot if possible

n/a

### Please provide the exception or error you saw

```true
The requested `code-example` reflects a new step added in #43500 to eliminate an error.
```

### Is this a browser-specific issue? If so, please specify the device, browser, and version.

```true
n/a
```

**Comment by gkalpak** [comment]
The necessary code already exists in [aio/content/examples/getting-started/src/app/app.module.ts][1], so this is just a matter of incorporating it into the guide. Closing this issue and continuing the discussion on #43500.

[1]: https://github.com/angular/angular/blob/a2eae02b44b04385e6439be363dfefac1cd997b2/aio/content/examples/getting-started/src/app/app.module.ts

**Comment by gkalpak** [comment]
@josmar-crwdstffng explained that we do not want to block #43500 with further changes and do those is a follow-up PR instead.

For reference:
I believe the fix in #43500 should look similar to #43229 (to keep the wording/formatting/code consistent with other parts of the tutorial).

**Comment by bob-watson** [comment]
Changed to P3 and closed because PR #43500 and its related PR #43646 appear to have resolved it.
